src/Plugin/EmulateElastic/ClusterKibanaHandler.php

```diff
@@ -0,0 +1,1401 @@
+<?php declare(strict_types=1);
+
+/*
+ Copyright (c) 2024, Manticore Software LTD (https://manticoresearch.com)
```

Comment:
To make future maintenance easier, I suggested using the notation "2024-present" or "year when the first file appeared-present." This will help us avoid updating the copyright every year. I think it's a good idea to follow this rule and adjust new files accordingly.

Reply:
Ok